src/test/java/io/rcktapp/rql/TestCollapse.java

```diff
@@ -1,17 +1,17 @@
 package io.rcktapp.rql;
```

Comment:
You don't need to worry about changes to the tests below - other than migrating to junit5 code, there were just a couple tests failing because their assertions were out of date with former code changes.